### ISSUE TITLE
Add/Edit Page Description

### ISSUE THREAD

**Issue by Omzig** [ironmansoftware/powershell-universal#2111]
### Steps to Reproduce

i am unable to set a description on the page

### Expected behavior

```console
Saved description would save
```

### Actual behavior

```console
saves nothing

Workaround:
edit the file directly in the configurations
```

### Environment data

PSU: 3.7.11

### Visuals

![image](https://user-images.githubusercontent.com/1987296/219505335-248a0438-25e0-4895-bffd-b294edf8c932.png)

**Comment by Omzig** [comment]
@adamdriscoll, can you fix this in 3.7.12?

**Comment by adamdriscoll** [comment]
Description is fixed in 3.7.12 and title is implemented in 3.8.0

**Comment by Omzig** [comment]
Related: https://github.com/ironmansoftware/issues/issues/2004
